tests/filecheck/transforms/convert_onnx_to_linalg.mlir

```diff
@@ -22,8 +22,8 @@
 // CHECK-NEXT:    } -> tensor<3x4xf64>
 
 %res_constant = "onnx.Constant"() {onnx_node_name = "/Constant", "value" = dense<1> : tensor<1xi64>}: () -> tensor<1xi64>
```

Comment:
can you please add a second constant with a different name and shape to the test?

Reply:
We need the constants to have different names, since they'll have different values. Can you please modify the lowering to make sure that the names are unique? My first thought is to reuse the onnx_node_name, but I'm not sure how that plays with the restriction on symbol names in MLIR. Another possible solution is `onnx_constant_1`, 2, 3, etc

Reply:
ahhh, my bad, i misunderstood, it should be sorted now :)